packages/react-mutation-mapper/src/component/mutationTable/DefaultMutationTable.tsx

```diff
@@ -151,6 +152,17 @@ export default class DefaultMutationTable extends React.Component<
                   );
     }
 
+    @computed
+    get signalAccessor() {
+        return this.hgvsgColumnDataStatus === 'pending'
```

Comment:
I change the name to `indexedVariantAnnotationDataStatus` because `hgvsg` and `hgvsc` are also using this function,  no need to create another duplication.